src/materials/ConcreteThermalMoisture.C

```diff
@@ -27,57 +27,70 @@ ConcreteThermalMoisture::validParams()
   params.addRequiredParam<std::string>(
       "type", "A string representing the Moose Object that is used to call this class");
 
-  // parameters for thermal properties calculations
-  MooseEnum thermal_conductivity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum thermal_capacity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum aggregate_type("Siliceous Carbonate", "Siliceous");
-
+  // // parameters which will be deprecated soon
+  MooseEnum thermal_capacity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004");
+  params.addDeprecatedParam<MooseEnum>("thermal_capacity_model",
+                                       thermal_capacity_model,
+                                       "thermal capacity models",
+                                       "Use thermal_model instead");
+  MooseEnum moisture_diffusivity_model("Bazant Xi Mensi");
+  params.addDeprecatedParam<MooseEnum>("moisture_diffusivity_model",
+                                       moisture_diffusivity_model,
+                                       "moisture diffusivity models",
+                                       "Use moisture_model instead");
+  params.addDeprecatedParam<Real>("ref_density_of_concrete",
+                                  2231.0,
+                                  "refernece density of porous media Kg/m^3",
+                                  "Use ref_density instead");
+  params.addDeprecatedParam<Real>("ref_specific_heat_of_concrete",
+                                  1100.0,
+                                  "reference specific heat of concrete J/Kg/0C",
+                                  "Use ref_specific_heat instead");
+  params.addDeprecatedParam<Real>("ref_thermal_conductivity_of_concrete",
+                                  3.0,
+                                  "reference thermal conductivity of concrete  W/m/0C",
+                                  "Use ref_thermal_conductivity instead");
+
+  // available model for thermal and moisture transport
+  MooseEnum thermal_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004");
```

Comment:
Add `thermal_conductivity_model` as a deprecated param too.

Reply:
I have done that now. I missed this one.